core/src/main/java/io/aiven/klaw/model/requests/TopicRequestModel.java

```diff
@@ -38,4 +38,6 @@ public class TopicRequestModel extends BaseRequestModel implements Serializable
   private Boolean deleteAssociatedSchema;
 
   private String otherParams;
+
+  private Integer topicId;
```

Comment:
Should this id be in the vase model as we will be expanding edit functionality to multiple resources?